include/dba.php

```diff
@@ -1299,37 +1299,29 @@ public static function buildCondition(array &$condition = [])
 						$condition_string .= " AND ";
 					}
 					if (is_array($value)) {
-						// Check if there are integer values in the parameters
+						/**
```

Comment:
Code standards: please don't use `/**` for comments because it is used for doxygen. Rather use `/*` or `/* *`